docs/source/terms_and_model.rst

```diff
@@ -400,34 +402,28 @@ referenced, and regardless of order. See
 Systemic Variation
 ##################
 
-Systemic Variation is a :ref:`Variation` of multiple
-molecules in the context of a system, e.g. a genome,
-sample, or homologous chromosomes.
-
-.. _Abundance:
-
-Abundance
-$$$$$$$$$
-
-*Abundance* is the measure of a quantity of a molecule
-in a system. :ref:`Copy Number <CopyNumber>` and
-gene expression variants are two common types of
-abundance variation, measuring the copies of a molecule
-present in a genome or expressed in a sample, respectively.
+A :ref:`Variation` of multiple molecules in the context of a system,
+e.g. a genome, sample, or homologous chromosomes.
 
 .. _CopyNumber:
 
 CopyNumber
-%%%%%%%%%%
-
-*Copy Number* captures the copies of a molecule within
-a genome, and can be used to express concepts such as
-amplification and copy loss.
+$$$$$$$$$$
+
+*Copy Number Variation* captures the copies of a molecule within a
+genome, and can be used to express concepts such as amplification
+and copy loss. Copy Number Variation has conflated meanings in the
+genomics community, and can mean either (or both) the notion of copy
+number *in a genome* or copy number *on a molecule*. VRS separates
+the concerns of these two types of statements; this concept is a type
+of :ref:`SystemicVariation` and so describes the number of copies in a
+genome. The related :ref:`MolecularVariation` concept can be expressed
+as an :ref:`Allele` with a :ref:`RepeatedSequenceExpression`.
```

Comment:
Just adding a comment here from somebody who is still learning about vrs.  Repeat expansion variants are different types of variants in my view from CNVs. Repeat expansion variants would mean that a repeated sequence got longer (e.g. FMR1/ fragile X syndrome).  CNVs could be tandem duplications, or interspersed duplications. Using short read sequencing and targeted enrichment, you might not be able to distinguish these two cases, since you might be unlucky and were not able to sequence the breakpoints. One might be just certain e.g. that an exon got duplicated (with uncertain breakpoints somewhere in the introns upstream and downstream).

Reply:
@andreasprlic I'm not sure I understand 100% as I am not a variant scientist or geneticist. We've gone around and around trying to determine the level of specificity we need to add to CNV dupes vs STRs. At this point in the model development we are providing our first version of the CopyNumber class which isn't meant to convey any notion of tandemness. We recognize that folks refer to Dupes (represented with HGVS notation) as a kind of CNV (sometimes, depending on size and possibly assay used to make the call). We struggle with converting HGVS dupe notation to a systemic CNV since it is technically meant to represent the number of copies on a single molecule. 

Our RepeatSequenceExpression is meant to be a tandem repeat and used inside an Allele which is a molecular variation subtype. 

So our struggle is whether it makes sense to represent and transform HGVS dupes into Alleles with RepeatSequenceExpressions thus presuming that HGVS dupes are ALWAYS tandem (when they are not necessarily) versus converting HGVS dupes to CopyNumber systemic variations and making a presumption about the absolute copy count conveyed by an HGVS dupe (problematic on sex chromosomes and not really assured in diploid chromosomes if the zygosity is not explicitly provided).  

The ambiguity that seems natural to experts that refer to CNV dupes is something of a mystery to me still which makes it nearly impossible to specify. 

If you see that the initial forms we are providing now are problematic, let's have a conversation, develop some examples that cover the gambit of possibilities and determine if we can support some or all of them now and provide guidance and wording in the spec to avoid significant difficulties as we evolve the spec.

If you could help by providing those real world examples, and letting us know when you can join a VRS meeting that would be helpful. Also, if you could let us know if you think this is a "show stopper" issue for releasing 1.2 that would be very important to know ASAP.

Thanks.

Reply:
Hi, Not all CNV dups are in tandem. In fact I recall hearing a story about an example where the difference in knowledge between a tandem DUP vs an interspersed DUP results in a different interpretation (pathogenic vs variant of unknown significance). 

I can try to dig out specific examples for this, if that is helpful. Perhaps a good next step is for me to find some examples and then we can try out how to represent them?

I'd be more than happy to join the next VRS meeting. I am now getting invites from the ga4gh mailer. I assume the invite would come from that?

Thanks

Reply:
yes. we are meeting this Monday at 12-1p ET to review the 1.2 PR and we should have time to discuss this point to make sure we can address any concerns around these issues. It would be ideal if you could attend and bring and example or two. Even if you can't gather the examples, we can discuss the concern properly with the group and determine what course, if any, needs to be taken.

Reply:
"CNV" has many meanings that differ in the size of the repeat unit and what is known about the location of the copies. A key message from the imminent 1.2 release is that we want to distinguish molecular changes, including repeats that we know are in tandem, from those that are systemic, where we know that they exist but not where. So, to be clear, we're not saying that all CNVs are tandem.

In my opinion, the many macroscropic meanings of CNVs -- from nucleotide and exonic tandem repeats to gene-scale duplications anywhere -- make it a difficult term to nail down as a single data structure.

The current direction in VRS is that we really have two kinds of CNVs: RepeatedSequenceExpression, which is a Molecular concept and implies tandemness, and CopyNumber, which is a Systemic concept that makes no assertion about where the copies are located. A CopyNumber even can be inferred from a RepeatedSequenceExpression, but not the converse.

Reply:
@larrybabb oh interesting. I did not get an invite for the meeting on Monday, but a notification that a meeting last week was canceled. I'd be glad to join, if somebody could forward an invite...

From what @reece says I think this would work. Here a few CNV variants that we could dig out (thanks @jinghuashi ): 

- here an imprecise CNV dup, where we don't know the breakpoints exactly, but an exon got duplicated, most likely in tandem and in-frame `NC_000017.10:g.41209048-?_41209172+?dup`, copy number is 3
<img width="1154" alt="Screen Shot 2021-06-24 at 2 10 51 PM" src="https://user-images.githubusercontent.com/187599/123333218-4df6df00-d4f6-11eb-9ca4-a88ad13b80e8.png">

- here another imprecise CNV dup, in tandem, but out of frame `NC_000013.10:g.32944519-?_32945257+?dup` , CN=3
<img width="1157" alt="Screen Shot 2021-06-24 at 2 16 48 PM" src="https://user-images.githubusercontent.com/187599/123333679-e68d5f00-d4f6-11eb-98be-cfb0840f6a34.png">

- An out of frame CNV dup, a precise event, results in a fusion protein `NC_000003.11:g.49161700_49165984dup`, also CN=3
<img width="1161" alt="Screen Shot 2021-06-24 at 2 24 39 PM" src="https://user-images.githubusercontent.com/187599/123334558-2143c700-d4f8-11eb-9a70-a44e2e8bcabd.png">

- For sex chromosomes we prob want to be able to express what is an expected copy number, and what is observed. For a male sample there would be only one copy of X (CNNORM=1), but the pseudo autosomal regions would have CNNORM=2, since they are homologous between X, Y. For a female sample X would have CNNORM=2.

- just to add, here a [reference](https://pubmed.ncbi.nlm.nih.gov/25640679/) about most CNV dups being in tandem, but not all.

As for repeat-expansion variants, for an FMR1 variant (linked to fragile X syndrome), I'd like to be able to say that in a sample we observed e.g. 94 repeats of CGG ([reference](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3151325/)). Something along the lines of `NC_000023.10:g.146993569-146993571[94]`

Reply:
great stuff @reece. stupid question (pardon my ignorance) but is the technology used to make these dup calls assured to know that these copied regions are in phase? If so, then these dups are really a kind of Molecular CNV and not really a Systemic CNV. Should we help the community clearly differentiate between systemic and molecular CNVs? 

Another question ... Are all dups considered a single extra copy or can there be more than one extra copy? 

In clinvar on this variant [GRCh38/hg38 1p34.3(chr1:35934217-36821999)x4](https://www.ncbi.nlm.nih.gov/clinvar/variation/149964/) it seems clear to me that there are 4 copies (somewhere in the genome) of the specified region.  Yet they also represent it down in the HGVS list as NC_000001.11:g.(?_35934217)_(36821999_?)dup.  So is this really a situation where we know there are 2 copies of the region on both instances of the chr1 molecule (in other words a homozygous dupe?). Again, apologies for my lack of knowledge, but getting to the bottom of all of these scenarios and how folks interpret these representations of variants is essential to getting our structures established so everyone can say exactly what they mean to say in a precisely specific or precisely ambigous way - whatever fits the needs of the producer.

Reply:
@larrybabb I think we generally try to be careful how we report such variants. In some cases we can actually detect if there is an interspersed duplication (which makes them more likely to be benign). In other cases the data suggests that it is likely a tandem duplication, but there is still some small degree of uncertainty so we try to express this using appropriate language. Does that mean it should be a Systemic CNV, rather than a molecular?

I think there can be more than one extra copy, such as your example which could be explained with a potential homozygous dup. Alternatively, a ref-alt heterozygous sample where the alt has 3 copies could also lead to a copy nr. of 4. I am not too familiar with the details of such large events, but I can imagine that there might be a few experimental challenges when trying to resolve the zygosity of such a sample.

Reply:
Is it possible to represent CNV delins events?

Reply:
Please describe a CNV delins. 

Suffice it to say that there's a lot that VRS can't represent *yet*. Use cases drive development, so let's discuss the use.

Reply:
Based on yesterday's discussion, I think this is resolved for now.